custom_components/kia_uvo/KiaUvoApiCA.py

```diff
@@ -109,6 +112,19 @@ def get_cached_vehicle_status(self, token: Token):
         vehicle_status = {}
         vehicle_status["vehicleStatus"] = response
         vehicle_status["vehicleStatus"]["time"] = response["lastStatusDate"]
+        service_status = {}
+        service_status = self._get_service_status(token)
```

Comment:
as of now, I am not following `_` (underscore naming pattern) ,you can use it without it.